src/components/Torrents.vue

```diff
@@ -122,15 +122,15 @@
             @click="editTracker"
             :title="$t('title.edit_tracker')"
           >
-            <v-icon>mdi-server</v-icon>
+            <v-icon>mdi-vector-polyline-edit</v-icon>
           </v-btn>
           <v-btn
             icon
             @click="recheckTorrents"
             :title="$t('recheck')"
             :disabled="selectedRows.length == 0"
           >
-            <v-icon>mdi-backup-restore</v-icon>
+            <v-icon>mdi-restore</v-icon>
```

Comment:
reject

Reply:
I believe this one is better than the `backup` one. The `backup` one is more of a `Get something from somewhere and copy it here` that's why it has the dot in the middle.